src/WorkspaceContext.ts

```diff
@@ -495,6 +495,17 @@ export class WorkspaceContext implements vscode.Disposable {
         }
     }
 
+    /**
+     * Return is folder is considered a valid root folder ie does it contain a SwiftPM
```

Comment:
Think this is a typo 
```suggestion
     * Return if folder is considered a valid root folder ie does it contain a SwiftPM
```